### PR TITLE
Fix 1.6 to master merge

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by asaunier** [comment]
Makes sense indeed :)

**Comment by pvalsecc** [comment]
It would be saner if you were on the chat. I was working on fixing the Alembic merge as well  :-(

**Comment by sbrunner** [comment]
Witch chat ?

**Comment by ochriste** [comment]
jabber

**Comment by sbrunner** [comment]
My network is not stable an I will be disconnected :-(

**Comment by pvalsecc** [comment]
Ship it!

**Comment by sbrunner** [comment]
Thanks :-)

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -286,7 +286,6 @@ TEMPLATES_FILES = $(shell find $(PACKAGE)/templates -type f -print)
 
 # Templates
 TEMPLATE_EXCLUDE += .build print/templates \
-	templatecache.js.mako \
```
Comment (elemoine) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Is this wanted?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Yes, if the file is named `templatecache.mako.js` as in 1.6, it's not needed :-)

Comment (elemoine) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: I don't understand. I thought `TEMPLATE_EXCLUDE` was a list a files that shouldn't be passed to `c2c.template`. And we don't want to pass `templatecache.js.mako` to `c2c.template`.

Am I not understanding this correctly?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Yes, if the file is named `templatecache.js.mako` but he is named `templatecache.mako.js` :-)

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: `.js` and `.mako` inversion!

Comment (elemoine) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Oh ok. A bit fragile but it works.

**Pull request merged**